TypeScriptReact.YAML-tmLanguage

```diff
@@ -94,16 +94,18 @@ repository:
 
   jsx-tag-without-attributes:
     name: meta.tag.without-attributes.tsx
-    begin: (<)\s*([_$a-zA-Z][-$\w.]*(?<!\.|-))\s*(>)
-    end: (</)\s*([_$a-zA-Z][-$\w.]*(?<!\.|-))\s*(>)
+    begin: (<)\s*(([a-z][a-z0-9]*|([_$a-zA-Z][-$\w.]*))(?<!\.|-))\s*(>)
+    end: (</)\s*(([a-z][a-z0-9]*|([_$a-zA-Z][-$\w.]*))(?<!\.|-))\s*(>)
```

Comment:
Can you please change expressions in this PR from 
```
(([a-z][a-z0-9]*|([_$a-zA-Z][-$\w.]*))(?<!\.|-))
```

to 
```
((?:[a-z][a-z0-9]*|([_$a-zA-Z][-$\w.]*))(?<!\.|-))
```

This will help in easy reading when reading the capture groups.